documentation/src/main/asciidoc/topics/proc_using_jgroups_default.adoc

```diff
@@ -0,0 +1,57 @@
+[id='jgroups_getting_started-{context}']
+= Getting Started with Default Stacks
+{brandname} uses JGroups protocol stacks so nodes can send each other messages on dedicated cluster channels.
+
+{brandname} provides preconfigured JGroups stacks for `UDP` and `TCP` protocols.
+You can use these default stacks as a starting point for building custom cluster transport configuration that is optimized for your network requirements.
+
+.Procedure
+
+. Locate the default JGroups stacks, `default-jgroups-*.xml`, in the `default-configs` directory inside the `infinispan-core-{ispn_version}.jar` file.
+ifdef::server_mode[]
++
+The `jar` file is in the `{server_home}/lib` directory.
++
+endif::server_mode[]
+. Do one of the following:
+
+* Use the `stack` attribute in your `infinispan.xml` file.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+include::config_examples/config_transport_stack.xml[]
+----
+<1> Uses `default-jgroups-udp.xml` for cluster transport.
+
+ifdef::embedded_mode[]
+* Use the `addProperty()` method to set the JGroups stack file:
++
+[source,java]
+----
+include::code_examples/ConfigureTransportDefault.java[]
+----
+<1> Uses the `default-jgroups-udp.xml` stack for cluster transport.
+endif::embedded_mode[]
+
+ifdef::server_mode[]
+* Use the `cluster-stack` argument when you start the server:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ bin/server.sh --cluster-stack=udp
```

Comment:
Hint: that will only work if the infinispan.xml contains the transport expression stack=${infinispan.cluster.stack:XXX}

Reply:
Is that correct? I thought `--cluster-stack` works for any of the default stacks.

Reply:
The Bootstrap class will check --cluster-stack and add a property Server.INFINISPAN_CLUSTER_STACK -> infinispan.cluster.stack.

So it woll replace the name of the stack udp/tcp or what ever customized stack you have in config when started
IF and only if the infinispan.xmk include the expression with that name, if you change the expression or remove it it will have no effect!

Reply:
Will address this in separate Jira as discussed.